### PR TITLE
Add DQM^2-Mirror k8 manifest

### PR DESCRIPTION
**Pull request by pmandrik**
Add DQM^2-Mirror k8 manifest for the first time.

### CODE REVIEW

File: kubernetes/cmsweb/services/dqmsquare.yaml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    metadata:
+      labels:
+        app: dqm-square-mirror-server
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      containers:
+      - name: dqm-square-mirror-server
+        image: registry.cern.ch/cmsweb/dqmsquare_mirror:latest #imagetag
```
Comment (muhammadimranfarooqi) [inline kubernetes/cmsweb/services/dqmsquare.yaml:39]: Please remove `:latest` from image

File: kubernetes/cmsweb/services/dqmsquare.yaml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    metadata:
+      labels:
+        app: dqm-square-mirror-parser
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      containers:
+      - name: dqm-square-mirror-parser
+        image: registry.cern.ch/cmsweb/dqmsquare_mirror:latest #imagetag
```
Comment (muhammadimranfarooqi) [inline kubernetes/cmsweb/services/dqmsquare.yaml:87]: Please remove `:latest` from image

File: kubernetes/cmsweb/services/dqmsquare.yaml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    metadata:
+      labels:
+        app: dqm-square-mirror-robber
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      containers:
+      - name: dqm-square-mirror-robber
+        image: registry.cern.ch/cmsweb/dqmsquare_mirror:latest #imagetag
```
Comment (muhammadimranfarooqi) [inline kubernetes/cmsweb/services/dqmsquare.yaml:128]: Please remove `:latest` from image

File: kubernetes/cmsweb/services/dqmsquare.yaml [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    metadata:
+      labels:
+        app: dqm-square-mirror-robber-oldruns
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      containers:
+      - name: dqm-square-mirror-robber-oldruns
+        image: registry.cern.ch/cmsweb/dqmsquare_mirror:latest #imagetag
```
Comment (muhammadimranfarooqi) [inline kubernetes/cmsweb/services/dqmsquare.yaml:174]: Please remove `:latest` from image

File: kubernetes/cmsweb/services/dqmsquare.yaml (hunk 1 above)
Comment (pmandrik) [reply kubernetes/cmsweb/services/dqmsquare.yaml]: done as well as for following occurrences

**Pull request merged**